### PR TITLE
zh/news/releases/1.9.x/announcing-1.9.2/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
zh/news/releases/1.9.x/announcing-1.9.2/index.md

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by yanrongshi** [comment]
@Xunzhuo  help to review,boss

**Comment by Xunzhuo** [comment]
I will reach it out today :)

File: content/zh/news/releases/1.9.x/announcing-1.9.2/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    - /zh/news/announcing-1.9.2
+---
+
+这个版本说明描述了 Istio 1.9.1 和 Istio 1.9.2 之间的区别。
+
+{{< relnote >}}
+
+## 改变{#changes}
+
+- **修复** 修复了一个问题。所以现在在 `EnvoyFilter` 中配置参数时，传输套接字的参数将会被包含在其中。
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.9.x/announcing-1.9.2/index.md:18]: remove 所以

File: content/zh/news/releases/1.9.x/announcing-1.9.2/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  ([Issue #28996](https://github.com/istio/istio/issues/28996))
+
+- **修复** 修复了在 `istiod` 中禁用默认入口控制器后导致日志泄露的问题。
+  ([Issue #31336](https://github.com/istio/istio/issues/31336))
+
+- **修复** 修正了一个问题，所以现在的 Kubernetes API 服务器默认情况下认为是存在于集群本地内。 这意味着任何试图访问 `kubernetes.default.svc` 的 pod 都
```
Comment (Xunzhuo) [inline content/zh/news/releases/1.9.x/announcing-1.9.2/index.md:24]: Ditto

**Review by Xunzhuo — CHANGES_REQUESTED** [review]
Small changes, other than LGTM.

**Review by Xunzhuo — APPROVED** [review]
Thanks @yanrongshi

**Pull request merged**